packages/amplify-provider-awscloudformation/lib/push-resources.js

```diff
@@ -83,6 +85,19 @@ async function run(context, category, resourceName) {
     });
 }
 
+function emitAmplifyMetrics(
+  context, resourcesToBeCreated,
+  resourcesToBeUpdated, resourcesToBeDeleted,
+) {
+  const metricPromises = [];
+
+  resourcesToBeCreated.forEach(resource => metricPromises.push(emitMetric(context, { action: 'create', service: resource.service })));
```

Comment:
Can we send all the events at once instead of sending events for each?

Reply:
We can, but I don't see a huge win over here..feel this is a cleaner implementation.

Reply:
ok